test/interface/mass_matrix_tests.jl

```diff
@@ -196,23 +197,28 @@ end
   @test sol1[end] ≈ sol13[end] atol=1e-9
 end
 
+function _norm_dsol2(alg,prob,prob2; kwargs...)
+  sol = solve(prob,  alg; kwargs...)
+  sol2 = solve(prob2,alg; kwargs...)
+  norm(sol[end] .- sol2[end])
+end
 @testset "Dependent Mass Matrix Tests" for mm in (dependent_M1, dependent_M2)
   # test each method for exactness
   for iip in (false, true)
     @show iip
     prob, prob2 = make_mm_probs(mm, Val(iip))
-
-    @test_broken _norm_dsol(ImplicitEuler(),prob,prob2) ≈ 0 atol=1e-7
-    @test _norm_dsol(RadauIIA5(),prob,prob2) ≈ 0 atol=1e-12
-    @test_broken _norm_dsol(ImplicitMidpoint(extrapolant = :constant),prob,prob2) ≈ 0 atol=1e-10
-
-    @test_broken _norm_dsol(QNDF1(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(QBDF1(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(QNDF2(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(QBDF2(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(ABDF2(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(QBDF(),prob,prob2) ≈ 0 atol=1e-7
-    @test_broken _norm_dsol(QNDF(),prob,prob2) ≈ 0 atol=1e-7
+    eulersol = solve(prob, ImplicitEuler(nlsolve=NLNewton(κ=1e-10)), reltol=1e-10)
+    @test _norm_dsol2(ImplicitEuler(nlsolve=NLNewton(κ=1e-10)),prob,prob2,reltol=1e-10) ≈ 0 atol=5e-4
+    @test _norm_dsol2(ImplicitMidpoint(nlsolve=NLNewton(κ=1e-10)),prob,prob2,tstops=eulersol.t) ≈ 0 atol=1e-6
+    @test_skip _norm_dsol(RadauIIA5(),prob,prob2) ≈ 0 atol=1e-12
```

Comment:
why skip this? wasn't it working?

Reply:
No, RadauIIA5 passed because it ignored the mass matrix completely, and `update_coefficients` didn't update the coefficients at all.

Reply:
oh... oh...